GameWorld/HexGrid/DrawableHexSphere.cs

```diff
@@ -114,6 +122,46 @@ public void SetGroundHeight(int groundHeight)
             InitializeVertices();
         }
 
+        public double GetHeightFromCoveragePercent(int coveragePercent)
+        {
+            double resultHeight = 0.5f;
+            List<double> tiles_heights = new List<double>();
+
+            foreach (var t in _sphereGrid.Tiles)
+            {
+                tiles_heights.Add(t.Height);
+            }
+
+            tiles_heights.Sort();
+
+            if (0 > coveragePercent || 100 < coveragePercent)
+            {
+                resultHeight = 0.5f;
```

Comment:
It's undefined, so I'm just using some default choice from my point of view. Can be changed in the future to whatever would seem more suitable. For now doesn't seem critical.